tests/EnumFromTest.php

```diff
@@ -2,81 +2,54 @@
 
 namespace Spatie\Enum\Tests;
 
-use PHPUnit\Framework\TestCase;
 use Spatie\Enum\Enum;
 use stdClass;
 use TypeError;
+//
+//test('from will throw type error for unallowed value types', function($value) {
+//    $this->expectException(TypeError::class);
+//
+//    HttpMethod::from($value);
+//});
 
-class EnumFromTest extends TestCase
-{
-    /**
-     * @test
-     * @dataProvider unallowedTypes
-     */
-    public function from_will_throw_type_error_for_unallowed_value_types($value)
-    {
-        $this->expectException(TypeError::class);
+test('from resolves all allowed value types', function() {
+    $this->assertTrue(HttpMethod::GET()->equals(HttpMethod::from('GET')));
```

Comment:
Could you convert all assertions to Pest's `expect()` API?

Reply:
Okay done now